pkg/network/cache/fake/infocache.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	"k8s.io/apimachinery/pkg/types"
 
-	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/network/cache"
+	"kubevirt.io/kubevirt/pkg/network/cache"
```

Comment:
what's the reasoning to put it in a different import group?

Reply:
the IDE did it, I just didn't notice.